### PR TITLE
Fix typo and double spaces for comments

### PR DESCRIPTION
**Pull request by ch33hau**
Fixed minor typo and removed double spaces from comments.

Thanks for reviewing it =)
(I have already signed the CLA)

### CODE REVIEW

File: client/influxdb.go [hunk 1]
```diff
@@ -189,7 +189,7 @@ func (c *Client) Query(q Query) (*Response, error) {
 
 // Write takes BatchPoints and allows for writing of multiple points with defaults
 // If successful, error is nil and Response is nil
-// If an error occurs, Response may contain additional information if populated.
+// If an error occurs, Response may contains additional information if populated.
```
Comment (otoolep) [inline client/influxdb.go]: "contain" is actually the correct word here. "contains" is incorrect.

File: client/influxdb.go [hunk 2]
```diff
@@ -263,7 +263,7 @@ func (c *Client) Write(bp BatchPoints) (*Response, error) {
 
 // WriteLineProtocol takes a string with line returns to delimit each write
 // If successful, error is nil and Response is nil
-// If an error occurs, Response may contain additional information if populated.
+// If an error occurs, Response may contains additional information if populated.
```
Comment (otoolep) [inline client/influxdb.go]: Ditto.

**Comment by otoolep** [comment]
Thanks @ch33hau -- however, not all the changes are necessary. If you update and squash, we can then merge.

File: client/influxdb.go (hunk 1 above)
Comment (otoolep) [inline client/influxdb.go]: This is because of the word "may".

**Comment by ch33hau** [comment]
@otoolep thanks for correcting me... I have updated it =)

**Pull request merged**